api/src/app.ts

```diff
@@ -59,7 +59,7 @@ export async function createApp(
 
   // JSON middleware to automatically parse incoming requests
   app.use(express.json());
-  // app.use((req, res, next) => { cookieParser()(req, res, next); });
+  app.use(cookieParser() as unknown as express.RequestHandler);
```

Comment:
right, I think I removed this in the express5 upgrade as I didn't see it used in the base app. Sorry.